content-repo/extra-docs/articles/paloaltonetworks-integrations-overview

```diff
@@ -0,0 +1,146 @@
+---
+id: PaloAltoNetworks_Integrations
+title: Palo Alto Networks Integrations Overview  
+description: The following maps the Palo Alto Networks Integrations and their use cases.
+---
+
+
+# Palo Alto Networks Integrations Overview
+Palo Alto Networks is a portfolio company with many different products and functionalities.
+Moreover, the APIs behind them are vast and do not fit under one integration.
+
+Review this document to determine the Palo Alto Networks integrations you need to fulfil your use cases.
+
+
+## [PAN-OS](https://xsoar.pan.dev/marketplace/details/PANOS)
+
+### Use Cases
+- Create and manage custom security rules in Palo Alto Networks PAN-OS.
+- Create and manage address objects, address-groups, custom URL categories, URL filtering objects.
+- Use the URL Filtering category information to enrich URLs.
+- Commit configurations to Palo Alto Firewall and to Panorama, and push configurations from Panorama to Pre-Defined Device-Groups of Firewalls.
+- Upgrade the version and content of the firewall.
+- Query the following PAN-OS log types: traffic, threat, url, data-filtering, and Wildfire.
+- Manage External Dynamic Lists (EDLs).
+
+### Playbooks
+- [PAN-OS Query Logs For Indicators](https://xsoar.pan.dev/docs/reference/playbooks/pan-os-query-logs-for-indicators)
+- [PAN-OS Commit COnfiguration](https://xsoar.pan.dev/docs/reference/playbooks/pan-os-commit-configuration)
+
+
+## [PAN-OS EDL Management](https://xsoar.pan.dev/marketplace/details/PaloAltoNetworks_PAN_OS_EDL_Management)
+
```

Comment:
Palo Alto Networks PAN-OS EDL Service is missing

Reply:
And EDL management is deprecated.

Reply:
correct, I will mention the edl service, and will rm edl management.